### ISSUE TITLE
Dashboard: Overlapping text in timeline when no experiments

### ISSUE THREAD

**Issue by shivanshs9** [chaos-mesh/chaos-mesh#1531]
## Bug Report

**What version of Kubernetes are you using?**
```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.7", GitCommit:"be3d344ed06bff7a4fc60656200a93c74f31f9a4", GitTreeState:"clean", BuildDate:"2020-02-11T19:34:02Z", GoVersion:"go1.13.6", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-30T20:19:45Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**What version of Chaos Mesh are you using?**
v1.1.1

**What did you do?**
Installed Chaos Mesh with dashboard, logged into Chaos Dashboard and found this issue at homepage.

**What did you expect to see?**
A simple text - "No experiments found"

**What did you see instead?**
![2021-03-03 13 28 34 localhost 7b82e503000c](https://user-images.githubusercontent.com/8217199/109775638-c411c600-7c27-11eb-9aa9-be1bbc9682e4.png)

**Comment by fewdan** [comment]
@g1eny0ung Looks like a display problem?

**Comment by g1eny0ung** [comment]
I'm already fixed this problem in 3a529c5777db1db44c6450d3028974737c19f5df. These will take effect after the next patch version is released.
